### ISSUE TITLE
Website Running Issue - Getting Started!

### ISSUE THREAD

**Issue by rowanc1** [ubcgif/appliedgeophysics.org#1]
- title: Geophysical Inversion Facility
- Buttons should be blue
- footer is dark blue
- switch image back to lack louise mesh
- split Publications + Presentations

**Comment by stevejpurves** [comment]
TODO:

- [ ] fix hero image overlay, make text more readable

**Comment by lheagy** [comment]
One other question... in the myst preview, I quite like that it shows the journal as the venue (e.g. as defined in the myst.yml https://github.com/ubcgif/2023-heagy-oldenburg-gji-casing-permeability/blob/07e25c967fff6e1319f68a513ac0008b55d4bf81/paper/myst.yml#L20-L22). Would it be possible to grab the venue from the myst.yml if defined?  

# MyST preview
![image](https://github.com/ubcgif/appliedgeophysics.org/assets/6361812/86e02b16-56e1-40d2-ac68-c86321b1e2a4)

# Curvenote site
![image](https://github.com/ubcgif/appliedgeophysics.org/assets/6361812/5eb4862a-4c2e-4c50-9cdf-d2d45410bf36)

**Comment by stevejpurves** [comment]
yes I think we can add an option to allow the myst.yml venue though
